pkgs/development/libraries/wxwidgets/wxGTK30.nix

```diff
@@ -1,29 +1,37 @@
 { lib
 , stdenv
+, expat
 , fetchFromGitHub
 , gst_all_1
+, withGtk2 ? true
```

Comment:
Can we drop this?

Reply:
`wxGTK30` defaults to `withGtk2 = true` in `pkgs/top-level/all-packages.nix`, so I guess it's not immediate.

Reply:
GTK2 is pretty EOL and already removed from many places. I would guess that wxGTK30 is used by accident and really we want to use the gtk3 variant in almost all places.

Reply:
In the meantime, I'm pretty sure that we can drop GTK2 from wxGTK31 after #194682.